iosApp/iosApp.xcodeproj/project.pbxproj

```diff
@@ -1551,7 +1551,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 16.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.4;
```

Comment:
spoke with product IRL after raising [in slack ](https://mbta.slack.com/archives/C062QNAJZ2M/p1727883523066669) - opting to bump the min version.